include/openssl/core_names.h

```diff
@@ -195,7 +205,13 @@ extern "C" {
 
 /* Key Exchange parameters */
 
-#define OSSL_EXCHANGE_PARAM_PAD      "pad" /* uint */
+#define OSSL_EXCHANGE_PARAM_PAD                   "pad" /* uint */
+#define OSSL_EXCHANGE_PARAM_EC_ECDH_COFACTOR_MODE "ecdh-cofactor-mode" /* int */
+#define OSSL_EXCHANGE_PARAM_KDF_TYPE              "kdf-type" /* int */ /* TODO(romen): magic numbers? */
```

Comment:
See magic numbers comment above.

This is currently an `int`, but there are potential clashes between magic values reserved for ECDH and magic values used for DH for this param.

Reply:
As per my other comment, I think you could use the "Known KDF names". That also allows the possibility of some new KDF that we've never heard of before being provided by an external provider.